miniscope_io/device_update.py

```diff
@@ -0,0 +1,134 @@
+import argparse
```

Comment:
This will need a bit of massaging to make general, since presumably we want to have a single entrypoint for all types of cameras like

`mio configure --port COM1 -c config.yaml` 

but I think fine for this PR. appreciate breaking this out to its own module!

Reply:
Sounds good. Related to the cle entry point issue, I'll start from making the streamDaq a bit more module-ish now it kind of works for a single function.